store/index.js

```diff
@@ -1,319 +1,344 @@
+		annotated: null
+	},
 
-			// Category paint brushes
-			current: {
-	
-				bColor: "white",
-				category: "",
-				fColor: "black"
-			},
-			default: {
-	
-				bColor: "white",
-				category: "",
-				fColor: "black"			
-			},
+    // Data dictionary (i.e. participants.json)
 
-			// Stores currently painted items
-			paintingData: {},
+	dataDictionary: {
 
-			// Stores currently loaded table data
-			tableData: {}
-		},
+		// File type of the original data dictionary file
+		fileType: "",		
 
-		// annotation.vue
-		annotation: {
+		// Original data dictionary file data
+		original: null,
 
-		},
+		// User-amended data dictionary file data
+		amended: {},	
+	},
 
-		// download.vue
-		download: {
+	// Stores table data in format ready for Bootstrap table
+	// This is an array of objects. See the mutation
+	// 'setupColumnToCategoryTable' for exact format
+	columnToCategoryMap: {
 
-		}
 	},
 
-	painting: {
-		
-		classes: {
-
-			paint0: "category-style-0",
-			paint1: "category-style-1",
-			paint2: "category-style-2",
-			paint3: "category-style-3",
-			paint4: "category-style-4"
-		},
-
-		palette: {
-			backgroundColors: [],
-			foregroundColors: []
-		}
-	}
+    // Hardcoded list of categories used on the categorization page
+    // and possibly elsewhere in the tool
+    categories: [],
+	
+	// This is a computed direct map between current categories and CSS classes
+	// See getter 'categoryClasses'
+	categoryClasses: null,
+
+	// The following fields are only accessed by store methods
+
+    // Maps our categories in 'categories' to colors in 'toolColorPalette'
+    // (Final class names pending). This way colors can be swappable and
+    // rearrangeable for categories.
+    categoryToColorMap: {},
+
+    // Map of the tools colors to CSS classes containing color (and possibly
+    // other style) values. More palettes could be defined here, either out of
+    // user preference or if we ever decided to code a light/dark mode feature
+    toolColorPalette: {
+
+        color1: "category-style-0",
+        color2: "category-style-1",
+        color3: "category-style-2",
+        color4: "category-style-3",
+        color5: "category-style-4",
+		colorDefault: "category-style-default"
+    }
 })
   
 // Actions - Call mutations to change state data in order to maintain trace of 
 // what component changed state data and when
 export const actions = {
 
-	// Landing page actions
+	// Initializations
 
-	saveTsvFile(p_context, p_tsvLines) {
+	createColumnToCategoryMap(p_context) {
 
-		// 1. Attempt to convert the tsv lines into a dict for each line if valid data given
-		let newTsvData = [];
-		if ( null != p_tsvLines ) 
-			newTsvData = convertTsvLinesToDict(p_tsvLines);
+		p_context.commit("setupColumnToCategoryMap");
+	},
 
-		// 2. Save either an empty array or array of tsv dictionaries to state data
-		p_context.commit("setTsvFile", newTsvData);
+	initializeCategories(p_context, p_categories) {
```

Comment:
This is actually a future function. `initializeCategories` will be called when we want to change category lists for different annotators. Since I was implementing dynamic category setup and creating a mutation (`setupCategories`) for this purpose, a corresponding action accessible by the pages (the one that will be swapping category lists) makes sense.